### PR TITLE
Update loglevels for redownload and consensus

### PR DESCRIPTION
**Pull request by mchrapek**

### CODE REVIEW

**Comment by kpudlik** [comment]
I would put more meaningful name to that PR, like:

"Update loglevels for p2p and consensus"

File: src/main/scala/org/constellation/consensus/ConsensusScheduler.scala [hunk 1]
```diff
@@ -25,10 +25,9 @@ class ConsensusScheduler(
     .void
 
   val crossTalkConsensus: IO[Unit] = consensusManager.startOwnConsensus().void.handleErrorWith {
-    case error: ConsensusStartError =>
-      IO(logger.debug(error.getMessage))
-    case error: ConsensusError => IO(logger.warn(error.getMessage))
-    case unexpected            => IO(logger.warn(unexpected.getMessage))
+    case error: ConsensusStartError => IO(logger.warn(error.getMessage))
+    case error: ConsensusError      => IO(logger.warn(error.getMessage))
+    case unexpected                 => IO(logger.warn(unexpected.getMessage))
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/consensus/ConsensusScheduler.scala]: Is warn the correct level? If it's en error and named as error, shouldn't it be `logger.error`?

Comment (mchrapek) [reply src/main/scala/org/constellation/consensus/ConsensusScheduler.scala]: I'm not sure if situations like:
- No active peers to start consensus
- No tips to start consensus 
- Node has already start own consensus
should be a `warn` or `error`? :thinking: 
What do you think @marcinwadon ?

Comment (marcinwadon) [reply src/main/scala/org/constellation/consensus/ConsensusScheduler.scala]: all those above should be `debug` then :) but `unexpected error` should be an `error` anyway

Comment (mchrapek) [reply src/main/scala/org/constellation/consensus/ConsensusScheduler.scala]: fixed

**Pull request merged**